javascript-forms/form-values.js

```diff
@@ -0,0 +1,17 @@
+
+var contactForm = document.querySelector('#contact-form');
+contactForm.addEventListener('submit', handleSubmit);
+
+function handleSubmit(event) {
+  event.preventDefault();
+  var name = contactForm.name.value;
+  var email = contactForm.email.value;
+  var message = contactForm.message.value;
```

Comment:
For the sake of consistency, we stick with reading the `elements` property of the form in order to get values.

```js
var name = contactForm.elements.name.value;
// etc
```